src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -373,6 +374,24 @@ public String getDartSdkPath() {
     return FlutterSdkUtil.pathToDartSdk(getHomePath());
   }
 
+  /**
+   * Return the existing 'flutter_tester' executable.
+   */
+  @Nullable
+  public VirtualFile getTester() {
+    final String platformString;
+    if (SystemInfo.isMac) {
+      platformString = "darwin-x64";
+    } else if (SystemInfo.isLinux) {
+      platformString = "linux-x64";
+    } else {
+      return null;
+    }
+
+    final String relativePath = "bin/cache/artifacts/engine/" + platformString + "/flutter_tester";
```

Comment:
Hm... If `flutter_tester` is basically a Dart VM, it should be possible just to give it a Dart file to execute, and it will care about finding any platform dills itself, shouldn't it?

Or we need a special way of running by first compiling code outside of VM into kernel, because we want to massage this kernel?